build/scripts/EnlistmentHelperFunctions.psm1

```diff
@@ -315,6 +315,15 @@ function Update-BCArtifactVersion {
 
     Write-Host "Latest BCArtifact URL: $latestArtifactUrl"
 
+    if ($currentArtifactUrl -notlike "https*") {
```

Comment:
No need for `$latestArtifactUrl = Get-LatestBCArtifactUrl -minimumVersion $currentVersion` then (line 314).

Reply:
Why not? 🤔 We still want to use the latestArtifactUrl when currentArtifactUrl matches "https*" i.e. in main. 

And in release branches I still use it to see if we should use '**bcinsider**/Sandbox/$currentVersion/base/latest' or '**bcartifacts**/Sandbox/$currentVersion/base/latest'. That way we should get a PR generated for releases/25.5 when we start pushing 25.5 artifacts to bcartifacts.

Reply:
Oh, yeah, no I understand better what's happening here.